Automattic/wp-calypso
# Stats Subscribers: Use react-queries to get data

Related to [#74403](https://github.com/Automattic/wp-calypso/issues/74403)

## Proposed Changes

* Extends redux store and uses react-queries to get the data
* Currently, it is pulling through mock data. future PRs will swap this to using the mock API, then eventually the real API once it's ready. 

## Testing Instructions

* Open the Calypso Live Branch. Navigate to the main stats dashboard page
* Append the following to the URL: `?flags=stats/subscribers-section`
* Scroll down to the card next to Videos and check that subscriber data is being pulled through

![image](https://user-images.githubusercontent.com/30754158/228327630-165addcf-3a72-4dca-91eb-4d61890fa2eb.png)


## Pre-merge Checklist
